physics/SFC_Models/Land/RUC/lsm_ruc.meta

```diff
@@ -320,6 +320,7 @@
   type = real
   kind = kind_phys
   intent = inout
+  optional = True
```

Comment:
@climbfuji @dustinswales The majority of variables that you add optional to, are not optional. For example smois - it is a state soil moisture variable.